lib/views/pages/home/content/map/post_map.dart

```diff
@@ -62,24 +64,46 @@ class PostMap extends ConsumerWidget {
       loading: () => (),
     );
 
+    final fab = FloatingActionButton(
+      onPressed: () {
+        // Center the camera on the user's position and follow
+        positionValue.whenData((geoPoint) {
+          LatLng userPosition = LatLng(geoPoint!.latitude, geoPoint.longitude);
+          mapNotifier.enableFollowUser();
+          mapNotifier.moveCamera(userPosition);
+        });
+      },
+      child: const Icon(Icons.my_location),
+    );
+
+    final googleMap = GoogleMap(
+      key: postMapKey,
+      mapType: MapType.normal,
+      myLocationButtonEnabled: false,
+      myLocationEnabled: true,
+      zoomGesturesEnabled: true,
+      zoomControlsEnabled: false,
+      scrollGesturesEnabled: true,
+      rotateGesturesEnabled: false,
+      tiltGesturesEnabled: false,
+      initialCameraPosition: CameraPosition(
+        target: mapInfo.initialLocation,
+        zoom: mapNotifier.initialZoom,
+      ),
+      circles: mapNotifier.circles,
+      markers: mapMarkersNotifier.markers,
+      onMapCreated: mapNotifier.onMapCreated,
+    );
+
     return Expanded(
-      child: GoogleMap(
-        key: postMapKey,
-        mapType: MapType.normal,
-        myLocationButtonEnabled: true,
-        myLocationEnabled: true,
-        zoomGesturesEnabled: true,
-        zoomControlsEnabled: true,
-        scrollGesturesEnabled: true,
-        rotateGesturesEnabled: false,
-        tiltGesturesEnabled: false,
-        initialCameraPosition: CameraPosition(
-          target: mapInfo.initialLocation,
-          zoom: initialZoomLevel,
+      child: Listener(
+        onPointerDown: (e) {
+          mapNotifier.disableFollowUser();
+        },
```

Comment:
I think we can use the arrow synthax here too :)
```suggestion
        onPointerDown: (_) => mapNotifier.disableFollowUser(),
```

Reply:
Fixed in 778fdb66e781ffde2faf8dbcd1a0759763741209